config/routes.rb

```diff
@@ -1,6 +1,12 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  namespace :admin do
```

Comment:
should we somehow make sure that only admins can see this route? (not sure if we are going to add an auth system or not)